broker/src/main/java/io/camunda/zeebe/broker/system/management/PartitionStatus.java

```diff
@@ -21,7 +21,7 @@ public final class PartitionStatus {
   private final ExporterPhase exporterPhase;
   private final Long exportedPosition;
 
-  private PartitionStatus(
+  PartitionStatus(
```

Comment:
:wrench: I think this could be a simple `record` because it is only a data holder and contains no logic. Then you could delete all of the getters and the `toString` implementation.